Lib/pathlib.py

```diff
@@ -851,8 +851,9 @@ def samefile(self, other_path):
         return os.path.samestat(st, other_st)
 
     def iterdir(self):
-        """Iterate over the files in this directory.  Does not yield any
-        result for the special paths '.' and '..'.
+        """Yields path objects of the directory contents.
```

Comment:
Nit: I'd prefer to keep the imperative mood here 

```suggestion
        """Yield path objects of the directory contents.
```